.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
+- [ ] Updated documentation
+- [ ] External dependency updates
+- [ ] Internal dependency updates (with link to diff)
+- [ ] Screenshots
+- [ ] Update to CHANGELOG.rst
+- [ ] Update to AUTHORS.rst
 
-- [ ] Have tests been written for the new code?
-- [ ] Has documentation been written/updated?
-- [ ] New dependencies (if any) added to requirements file
-- [ ] Add an entry to CHANGELOG.rst
-- [ ] Add yourself it AUTHORS.rst if you don't appear there
+# Details
 
-## Reviewer guidance
+### Summary
 
-*If you PR has a significant size, give the reviewer some helpful remarks*
+description of the change, including screenshots when applicable
 
-## Issues addressed
+### Reviewer guidance
 
-List the issues solved or partly solved by the PR
+description of how to test the changes
 
-## Documentation
+### References
 
-If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)
-
-## Screenshots (if appropriate)
-
-They're nice. :)
+* related issues and PRs
+* related trello links
+* related mockups or specs
+* links to dependency updates
```

Comment:
The above 3 lines are all internal stuff

Reply:
I'm still suggesting to remove these 3 lines because they're internal LE stuff -- it's so we steer the project towards a public domain where community doesn't get confused or overloaded with things that are hard to access (complicated, unfamiliar, needs account access etc)

Reply:
okay I see what you're saying - we want to avoid confusing people who aren't familiar with these things. I think only the [trello roadmap board](https://trello.com/b/3I5QpMej/kolibri-feature-deadlines) is private though - the designs and external dependencies are public:

* https://learningequality.gitbooks.io/design-iterations/
* https://github.com/learningequality/iceqube
* https://github.com/learningequality/kolibri-exercise-perseus-plugin

I'll take a pass and try to make the language more inviting to community PRs while still providing proper guidance to LE contributors